standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/conf/TestMetastoreConf.java

```diff
@@ -510,4 +510,29 @@ public void testClassNames() {
     Assert.assertEquals(MetastoreConf.ACID_OPEN_TXNS_COUNTER_SERVICE_CLASS,
         AcidOpenTxnsCounterService.class.getName());
   }
+
+  /**
+   * Verify that getVar(Configuration conf, ConfVars var) method of MetaStoreConf.java runs as expected in case of unset environment variables
+   * or any other variables that don't have default Value type of String
+   */
+  @Test
+  public void testGetVarForUnsetSystemProperties(){
+    conf = MetastoreConf.newMetastoreConf();
+    System.clearProperty(ConfVars.USE_SSL.getVarname());
```

Comment:
Why are you doing this `` System.clearProperty``, we don't do that, play with the conf, maybe do a conf.set(ConfVars.USE_SSL.getVarname(), null); or something like that

Reply:
I was using `System.clearProperty()` to replicate the required use case considering it being used in many other unit tests also, but now have changed it to `conf.unset()` .